components/medical/fn_isInStableCondition.sqf

```diff
@@ -0,0 +1,39 @@
+
+
+params ["_unit"];
+
+// Hard-coding variables that would otherwise be in CBA and/or I can't seem to retrieve (defaultBloodVolume. These should be safe to hard-code.
+private _minHeartRatePAK = 40;
+private _minBloodPAK = 0.85;
+private _defaultBloodVolume = 6;
+
+
+// If the unit is dead, unconscious, bleeding, or below minimum heart rate, we cannot PAK.
+if (!alive _unit
+    || {(_unit getVariable ["ACE_isUnconscious", false])}
+    || {(_unit getVariable ["ace_medical_woundBleeding", 0]) > 0}
+    || {(_unit getVariable ["ace_medical_heartRate", 0]) < _minHeartRatePAK}
+) exitWith {false};
+
+// If they have insufficient blood, we cannot PAK.
+private _requiredBloodVolume = _defaultBloodVolume * (_minBloodPAK / 100);
+if ((_unit getVariable ["ace_medical_bloodVolume", 0]) < _requiredBloodVolume) exitWith {false};
+
+// Unit must not have fractures
+private _fractures = _unit getVariable ["ace_medical_fractures", []];
+if ((_fractures findIf {_x isEqualTo 1}) != -1) exitWith {false};
```

Comment:
Prevents PAK if there is an un-splinted fracture (value of 1) but allows it for splinted or no fracture (-1, 0 respective).